deploy/scripts/src/cri-lime/test.ts

```diff
@@ -38,23 +38,22 @@ const profiles: ProfileList = {
       preAllocatedVUs: 1,
       maxVUs: 1,
       stages: [
-        { target: 1, duration: '10s' } // Ramps up to target load
+        { target: 1, duration: '60s' } // Ramps up to target load
       ],
       exec: 'passportScenario'
     }
-
   },
-  load: {
+  lowVolumeTest: {
     fraudScenario1: {
       executor: 'ramping-arrival-rate',
       startRate: 1,
       timeUnit: '1s',
       preAllocatedVUs: 1,
-      maxVUs: 350,
+      maxVUs: 900,
       stages: [
-        { target: 10, duration: '30m' }, // Ramp up to 10 iterations per second in 10 minutes
-        { target: 10, duration: '15m' }, // Steady State of 15 minutes at the ramp up load i.e 10 iterations/second
-        { target: 0, duration: '5m' } // Ramp down duration of 5 minutes.
+        { target: 30, duration: '5m' }, // Ramp up to 30 iterations per second in 5 minutes
```

Comment:
This ramp up is faster than the `stress` scenario of up to 63/s over 15 mins. I'd increase the ramp up time here

Reply:
Done - Updated wherever applicable to ensure the ramp up of low volume tests is not faster than the stress scenario.